build.gradle

```diff
@@ -256,6 +256,7 @@ dependencies {
     testCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.1.0'
     testCompile group: 'com.xebialabs.restito', name: 'restito', version: '0.9.3'
     testCompile 'io.github.openfeign:feign-jackson:9.7.0'
+    testCompile 'pl.allegro.tech:embedded-elasticsearch:2.7.0'
```

Comment:
Not sure how good that embedded one. I prefer test containers in general https://www.testcontainers.org/modules/elasticsearch/